lib/OpenQA/Task/Git/Clone.pm

```diff
@@ -0,0 +1,132 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::Task::Git::Clone;
+use Mojo::Base 'Mojolicious::Plugin', -signatures;
+use Mojo::Util 'trim';
+
+use OpenQA::Utils qw(run_cmd_with_log_return_error);
+use Cwd 'abs_path';
+use Time::Seconds 'ONE_HOUR';
+
+sub register ($self, $app, @) {
+    $app->minion->add_task(git_clone => \&_git_clone);
+}
+
+
+# $clones is a hashref with paths as keys and urls to git repos as values.
+# The urls may also refer to a branch via the url fragment.
+# If no branch is set, the default branch of the remote (if target path doesn't exist yet)
+# or local checkout is used.
+# If the target path already exists, an update of the specified branch is performed,
+# if the url matches the origin remote.
```

Comment:
Is this really still the case? The code reads like this is only done if the target path does *not* exist yet (which itself is supposedly fine).

Reply:
> Is this really still the case? 

yes.